kernel/common/enumerable.rb

```diff
@@ -567,86 +567,110 @@ def first(n=undefined)
     nil
   end
 
-  def min
-    min = undefined
-    each do
-      o = Rubinius.single_block_arg
```

Comment:
What is `o` supposed to mean in this case? Can do get rid of the single letter variables?